notebooks/machine_learning/raw/ex8.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise: Classification\n",
+    "\n",
+    "Here you'll get some experience training a classification model yourself. What you'll do here is create a model that can determine if radio signals come from a pulsar. Pulsars are a rare type of neutron stars that produce radio signals we can detect on Earth. As the pulsars rotate, the beam of radio waves points directly at us, then moves away. This leads to a periodic signal that we can use to determine if the radio signal is actually from a pulsar or just noise.\n",
+    "\n",
+    "From the [dataset page on Kaggle](https://www.kaggle.com/pavanraj159/predicting-a-pulsar-star):\n",
```

Comment:
Most users just want to learn the ML. So I think it's plenty of detail alraedy in the intro paragraph. We don't need to say it again in the dataset page.

Broadly speaking, I worry a lot about the fact in UX design that users hate to read. So we should think of words as having some cost, and use them only if they move the user's knowledge forward (or they're necessary for the exercise to make sense).